{"requests/exceptions.py": [25, 26, 27, 28, 29, 30, 31, 32, 33, 34], "requests/models.py": [29, 30, 31, 32, 33, 34, 35, 466, 467, 468, 469, 470, 471, 472, 473, 474, 475, 476, 477], "tests/test_requests.py": [24, 25, 26, 27, 28, 29, 30, 2566, 2567, 2568, 2569, 2570, 2571, 2572, 2573, 2573]}